exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
+let myName = "Hayat";
+let nameLength = myName.length;
+
+console.log("My name is " + myName + " and my name is " + nameLength + " characters long"); // Logs 5
```

Comment:
Nice way to concatenate. However, you may try using interpolation. This may help to simplify your work.